modules/common/test/functional/job_test.go

```diff
@@ -389,3 +389,39 @@ var _ = Describe("job.Job", func() {
 		Expect(j.GetHash()).NotTo(Equal(oldHash))
 	})
 })
+
+var _ = Describe("job package", func() {
+	var namespace string
+
+	BeforeEach(func() {
+		// NOTE(gibi): We need to create a unique namespace for each test run
+		// as namespaces cannot be deleted in a locally running envtest. See
+		// https://book.kubebuilder.io/reference/envtest.html#namespace-usage-limitation
+		namespace = uuid.New().String()
+		th.CreateNamespace(namespace)
+		// We still request the delete of the Namespace to properly cleanup if
+		// we run the test in an existing cluster.
+		DeferCleanup(th.DeleteNamespace, namespace)
+	})
+
+	It("DeleteJob deletes existing jobs", func() {
```

Comment:
just curious if there is a reason why you haven't added these tests to the above Describe?

Reply:
Technically I can add it without any change. I just looked at the existing Describe and it was talking about the job.Job struct. While DeleteJob is does not have a job.Job receiver. But I guess it is less noise if I merge the two describe and just rename the one Describe to talk about the whole package.

Reply:
Will fix it in a minute.

Reply:
Done.